src/sdk/object_sdk.js

```diff
@@ -1107,8 +1109,20 @@ class ObjectSDK {
         this._check_is_readonly_namespace(ns);
         return ns.put_object_acl(params, this);
     }
+
+    is_nsfs_containerized_user_anonymous(token) {
+        return !token && !process.env.NC_NSFS_NO_DB_ENV;
+    }
+
+    is_nsfs_non_containerized_user_anonymous(token) {
+        return !token && process.env.NC_NSFS_NO_DB_ENV;
+    }
```

Comment:
I meant that these will be calls to bucketspace, so that in the bucketspace_fs level this is clearly going to allowed, but in bucketspace_nb it will not